docs/visual-basic/getting-started/index.md

```diff
@@ -28,12 +29,9 @@ Provides a list of Web sites and newsgroups that can help you find answers to co
 
 ## See also
 
-- [Get Visual Basic](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs)\
+- [Get Visual Basic](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019)\
```

Comment:
we've been removing the \ and adding back the two spaces because of loc issues
```suggestion
- [Get Visual Basic](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019)  
```

Reply:
:( :(

Reply:
This means one can't have the trim trailing whitespace setting turned on in VS Code. What's the loc issue?